apps/muebel_splunk_all_indexes/local/indexes.conf

```
@@ -175,3 +175,6 @@ frozenTimePeriodInSecs = 864001
 
 [this_is_a_new_index]
 frozenTimePeriodInSecs = 86401
+
+[newtork_data]
```

```suggestion
[network_data]
```
Looks like this might be a typo